README.md

```diff
@@ -599,6 +599,12 @@ breakpoint, the runner will open its own debugging console in the terminal that
 controls how tests are run. You can type "restart" there to re-run your test as
 many times as you need to figure out what's going on.
 
+Normally, browser tests are run in hidden iframes. However, when debugging, the
+iframe for the current test suite is expanded to fill the browser window so you
+can see and interact with any HTML it renders. Note that the Dart animation may
+still be visible behind the iframe; to hide it, just add a `background-color` to
+the page's HTML.
```

Comment:
I want it to be visible by default. This maximizes the frame for *all* browser tests, not just those that have HTML elements, and I want the spinner to be visible for the suites that don't.